p4-16/psa/PSA.mdk

```diff
@@ -573,15 +580,15 @@ A similar FIFO property is expected to hold for multicast packets,
 i.e. those that follow the "Make 0 or more copies" path in the
 pseudocode above.  By a short period of time after the control plane
 has stopped modifying the set of members of multicast groups, they
-should be 'stable', and all packets with the same `(ingress_port,
-ostd.multicast_group, ostd.class_of_service, egress_port)` should pass
-through the system in FIFO order, and be processed in the `Egress`
-control block in the same relative order that they were processed in
-the `Ingress` control block.
+should be 'stable', and all packets with the same `(ingress_port,`
+`ostd.multicast_group,` `ostd.class_of_service,` `egress_port)` should
```

Comment:
We either quote the entire parentheses, or individual names. Commas should not be quoted: `ostd.multicast_group`, `ostd.class_of_service`, `egress_port`)

Reply:
I had the whole expression quoted before, but that seems to cause Madoko to put the entire thing on one line, with no line breaks allowed.  What would you prefer?

Reply:
the individual fields without comma and parentheses, just as I put in the comment. oh, well, i guess i should have used some sort of raw input, because github interpreted the quotes :)

Reply:
I will soon include that change with another commit on this PR.  The PDF output from Madoko still looks reasonably formatted with that change.